galleries/examples/specialty_plots/mri_with_eeg.py

```diff
@@ -11,26 +11,26 @@
 import numpy as np
 
 import matplotlib.cbook as cbook
-import matplotlib.cm as cm
-from matplotlib.collections import LineCollection
 
-fig = plt.figure("MRI_with_EEG")
+fig, axd = plt.subplot_mosaic([
+    ["image", "density"],
+    ["EEG", "EEG"]
+], layout="constrained", width_ratios=[1.05, 2])
```

Comment:
This looks nice.  

I wonder if the 1.05 here needs to be explained a bit.  People may be surprised that they have to twiddle this by hand to get the aspect=1 image to align reasonably with the other two panels.  Conversely, you could _not_ twiddle it, and make `width_ratios=[1, 2]` and let the image look a bit misaligned.